cocoa/tests_backend/widgets/activityindicator.py

```diff
@@ -6,8 +6,8 @@
 class ActivityIndicatorProbe(SimpleProbe):
     native_class = NSProgressIndicator
 
-    def assert_is_hidden(self, value):
+    def assert_spinner_is_hidden(self, value):
         # Check that we're not displaying the widget when stopped.
         assert not self.native.isDisplayedWhenStopped()
         # No other assertion is possible, as there's no way to verify
-        # if the underlying widget is visible.
+        # if the underlying spinner is visible.
```

Comment:
This isn't an especially robust test in itself; we should probably add a little more in here.